seatunnel-e2e/seatunnel-connector-v2-e2e/connector-file-sftp-e2e/src/test/resources/xml/e2e.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why is this file needed? You can first create xml through sink and then read xml through source

Reply:
At first, I was using this file for source, but later I found out that I could generate it using fake source, and then read it again. I did not delete the file.